.eleventy.cjs

```diff
@@ -18,12 +16,6 @@ module.exports = config => {
 		excerpt_separator: "<!-- more -->"
 	});
 
-	config.setLibrary("md", markdownIt({
-			html: true,
-		})
-		.disable("code")
-	);
```

Comment:
This is now the default setting.

https://www.11ty.dev/docs/languages/markdown/#default-options